File: Four/PublicServiceVehicleLicense.php
<?php
include_once("License.php");

class PublicServiceVehicleLicense extends License {
    public function __construct() {
        parent::__construct("Public service veicles", 1900000, 4);
    }

    public function calculateTotalCost(): float {
        return $this->getCost(); // The total cost for the public service vehicle license is the base cost
    }
}
?>

File: java/Four/Four/PublicServiceVehicleLicense.java
public class PublicServiceVehicleLicense extends License {
    public PublicServiceVehicleLicense() {
        super("Public service vehicles", 1900000.00, 4);
    }

    @Override
    public double calculateTotalCost() {
        return this.cost; // The total cost for the public service vehicle license is the base cost
    }
}